packages/@okta/vuepress-site/docs/guides/saml-inline-hook/main/index.md

```diff
@@ -0,0 +1,327 @@
+---
+title: SAML assertion inline hook
+excerpt: Learn how to easily implement a SAML assertion inline hook
+layout: Guides
+---
+
+This guide provides a working example of an Okta SAML assertion inline hook. It uses the website [Glitch.com](https://glitch.com) to act as an external service to receive and respond to SAML assertion inline hook calls.
+
+---
+
+**Learning outcomes**
+
+* Understand the Okta SAML assertion inline hook calls and responses.
+* Implement a simple working example of a SAML assertion inline hook with a Glitch.com external service.
```

Comment:
it might help orient the reader to mention near the top of the guide that the Glitch project is in Node.js

Reply:
Added as part of the the Glitch bullet.